canary_tests.sh

```diff
@@ -19,16 +19,18 @@ pushd "$this_dir" > /dev/null
 
 systemctl stop viam-canary
 
-# It's possible that someone like Alan or Olivia was working on this board earlier in the day,
-# forgot to clean up what they were doing, and now the relevant pins are in use and can't also be
-# used by the canary tests. Stop what that was, so we don't get spurious test failures.
-systemctl stop viam-server
-
 # Install the latest rdk. Installing instead of upgrading to ensure we get the latest version and don't get stuck on stable.
 curl "https://storage.googleapis.com/packages.viam.com/apps/viam-server/viam-server-latest-$(uname -m).AppImage" -o viam-server
 chmod 755 viam-server
 ./viam-server --aix-install
 
+# It's possible that the config for the "normal" RDK on this board has components attached to it
+# because someone was working on that board recently. When we re-install the RDK on the previous
+# line, it restarts the "normal" RDK, even if it was previously stopped. This might take ownership
+# of the pins we want to test, so stop it (again) before starting the canary, or we'll get spurious
```

Comment:
wdym by stop it (again)? This seems like the only time it is stopped since were deleting the stop on line 25